doc/release_instructions.md

```diff
@@ -78,7 +78,7 @@ If you're doing a custom release (off a branch that isn't master), [skip to the
     > git push origin master
     ```
 
-8) Regenerate the golden deployment manifests: `./etc/testing/deploy-manifests/validate.sh --regenerate`
+8) Regenerate the golden deployment manifests: `make regenerate-test-deploy-manifests`
```

Comment:
You have to pass VERSION_ADDITIONAL=-rc1 here?

Reply:
The reverse, we need to not set `VERSION_ADDITIONAL` at all. If `VERSION_ADDITIONAL` is set, then the golden manifests will contain whatever string is in `VERSION_ADDITIONAL`, but travis-generated manifests will not and the manifest tests will fail.

Reply:
Oh wait I think you mean the exact opposite -- we should not pass VERSION_ADDITIONAL. That way the traget that get's built is only with 1.10.0 so travis-ci is happy. Is that's the case we should pass VERSION_ADDITIONAL=

Else the make file would take the commit it from the HEAD as VERSION_ADDITIONAL

Reply:
We actually don't need to set `VERSION_ADDITIONAL` to the empty string either (and travis doesn't when generating the test manifests—it doesn't set `VERSION_ADDITIONAL` at all), because `etc/testing/deploy-manifests/validate.sh` strips the commit hash from generated manifest files.

In general, my inclination would be to avoid setting `VERSION_ADDITIONAL` when generating the golden manifests, just because Travis doesn't set it for the test manifests

Reply:
So I am not sure how would calling make target result in a different set of manifests than what we did with rc5, in which we directly called validate script.

Reply:
The make target has `install` as a make dependency, which causes it to rebuild `pachctl`. 

Because this line in the release_instructions doesn't set `VERSION_ADDITIONAL`, the dependency on `install` will rebuild `pachctl` with `VERSION_ADDITIONAL` unset (so it'll take the default value of the current commit hash). Then, `validate.sh` will generate new golden deploy manifests that include the commit hash, and strip out the commit hash with a regex.

Finally, when we run the deploy tests in travis, travis will similarly build pachctl with `VERSION_ADDITIONAL` unset, and then run `validate.sh`, which will generate test deploy manifests that include the commit hash, and strip the commit hash with a regex. If nothing changed in `pachctl deploy`, these should be identical to the golden deploy manifests because they were generated the same way